kubernetes/base/05-registry-init.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: batch/v1
```

Comment:
To understand it correctly, this is used to setup the registry for the controller, and bring it close to the hosted experience.  

Should we mention in the [README](https://github.com/apigee/registry-experimental/blob/main/kubernetes/README.md) somewhere that this is setting up the server + controller?

Reply:
Thanks for the doc.